components/brave_extension/extension/brave_extension/webstore.ts

```diff
@@ -0,0 +1,18 @@
+const config = { attributes: true, childList: true, subtree: true }
+
+const callback = (mutationsList: MutationRecord[], observer: MutationObserver) => {
+  const button: Element = document.getElementsByClassName('g-c-R webstore-test-button-label')[0]
+
+  if (document.contains(button)) {
```

Comment:
minor nit

document will always contain button if found
why not
`if (button)`